lib/date_time/date_time.c

```diff
@@ -75,11 +75,7 @@ K_SEM_DEFINE(time_fetch_sem, 0, 1);
 static void date_time_handler(struct k_work *work);
 K_WORK_DELAYABLE_DEFINE(time_work, date_time_handler);
 
-static struct time_aux {
-	int64_t date_time_utc;
-	int64_t last_date_time_update;
-} time_aux;
-
+int64_t date_time_last_update_uptime;
 static bool initial_valid_time;
```

Comment:
I'm planning to remove `initial_valid_time` and test if `date_time_last_update_uptime != 0` (earlier `time_aux.last_date_time_update != 0`) is doing the same thing. We have function `date_time_is_valid()` which should be called when querying the state instead of using `initial_valid_time` directly.

Reply:
`initial_valid_time` removed